tools/python/boutiques/__init__.py

```diff
@@ -0,0 +1,10 @@
+from __future__ import absolute_import
+
+from .localExec import LocalExecutor
+from .invocationSchemaHandler import generateInvocationSchema, writeSchema
+from .validator import validate_json
+
+__all__ = ['localExec', 'invocationSchemaHandler', 'validator']
+
+version="0.0.1"
```

Comment:
Shouldn't this be the next tag in the repository? (I guess it would be 0.4.1, the schema being still 0.4).